pylcm/__init__.py

```diff
@@ -64,35 +70,48 @@ def msg_to_dict (data, e_channel, msg, statusMsg, verbose=False, lcm_timestamp=-
         # Iterate each constant of the LCM message
         constants = msg_getconstants (msg)
         for i in range(len(constants)):
-            myValue = None
-            myValue = eval('msg.' + constants[i])
-            data[e_channel][constants[i][:31]] = myValue
+            my_value = None
+            my_value = eval('msg.' + constants[i])
+            data[e_channel][constants[i][:31]] = my_value
 
     # Get lcm fields and constants
     fields = msg_getfields (msg)
 
     # Iterate each field of the LCM message
     for i in range(len(fields)):
-        myValue = None
-        myValue = eval(' msg.' + fields[i])
-        if (isinstance(myValue,int)     or
-            isinstance(myValue,long)    or 
-            isinstance(myValue,float)   or 
-            isinstance(myValue,tuple)   or 
-            isinstance(myValue,unicode) or 
-            isinstance(myValue,str)):
+        my_value = None
+        my_value = eval(' msg.' + fields[i])
+        if (isinstance(my_value,int)     or
+            isinstance(my_value,long)    or 
+            isinstance(my_value,float)   or 
+            isinstance(my_value,tuple)   or 
+            isinstance(my_value,unicode) or 
+            isinstance(my_value,str)):
             try:
-                data[e_channel][fields[i][:31]].append(myValue)
+                data[e_channel][fields[i][:31]].append(my_value)
             except KeyError as AttributeError:
-                data[e_channel][fields[i][:31]] = [(myValue)]
-
-        elif (hasattr(myValue,'__slots__')):
+                data[e_channel][fields[i][:31]] = [(my_value)]
+                
+        elif (hasattr(my_value,'__slots__')):
             submsg = eval('msg.' + fields[i])
-            msg_to_dict (data[e_channel], fields[i][:31], submsg, statusMsg, verbose)
-
+            msg_to_dict (data[e_channel], fields[i][:31], submsg, status_msg, verbose)
+ 
+        # Handles getting RBGD data from 'images' field
+        elif (fields[i] == "images" and 
+              isinstance(my_value, list) and 
+              isinstance(my_value[0], image_t)):
+            # Read image_t.data to numpy arrays
+            color_image = np.array(imageio.imread(my_value[0].data))
+            #depth_image = np.array(imageio.imread(my_value[1].data))
```

Comment:
how long is `my_value`? is `my_value[1]` a depth image? can you read it?

Reply:
`my_value` has len=2, one rgb and one depth image.  `my_value[1]` is a depth image, but reading it the same way gives an error.  Don't now how to properly read it.